SourceryTests/Models/TypeSpec.swift

```diff
@@ -199,3 +214,9 @@ class TypeSpec: QuickSpec {
         }
     }
 }
+
+private extension Sequence where Element == Import {
+    func sortedByPath() -> [Import] {
+        return sorted { $0.path < $1.path }
+    }
+}
```

Comment:
I don't think that's a useful extension, we could use `sorted` with closure as is
```suggestion
```

Reply:
done